{
  "Header": {
    "Original": "CVE-2012-2134 (The handle_connection_error function in ldap_helper.c in bind-dyndb-ld ...)",
    "Line": 546674,
    "ID": "CVE-2012-2134",
    "Description": "(The handle_connection_error function in ldap_helper.c in bind-dyndb-ld ...)"
  },
  "Annotations": [
    {
      "Original": "- bind-dyndb-ldap \u003cnot-affected\u003e (Fixed before initial upload to Debian)",
      "Line": 546675,
      "Type": "package",
      "Package": "bind-dyndb-ldap",
      "Kind": "not-affected",
      "Description": "Fixed before initial upload to Debian"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=815846",
      "Line": 546676,
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=815846"
    },
    {
      "Original": "NOTE: https://www.openwall.com/lists/oss-security/2012/04/24/14",
      "Line": 546677,
      "Type": "NOTE",
      "Description": "https://www.openwall.com/lists/oss-security/2012/04/24/14"
    },
    {
      "Original": "NOTE: https://pagure.io/bind-dyndb-ldap/c/cd33194c5a61e98cba53212458cce02b849077ba (v1.1.0rc1)",
      "Line": 546678,
      "Type": "NOTE",
      "Description": "https://pagure.io/bind-dyndb-ldap/c/cd33194c5a61e98cba53212458cce02b849077ba (v1.1.0rc1)"
    }
  ]
}